packages/contentstack-branches/src/utils/interactive.ts

```diff
@@ -148,6 +150,28 @@ export async function selectMergeExecution(): Promise<string> {
   return strategy;
 }
 
+export async function selectContentMergePreference(): Promise<string> {
+  const strategy = await cliux
+    .inquire({
+      type: 'list',
+      name: 'module',
+      choices: [
+        { name: 'Both existing and new', value: 'existing_new' },
+        { name: 'New only', value: 'new' },
+        { name: 'Existing only', value: 'existing' },
+        { name: 'Ask for preference', value: 'ask_preference' },
+      ],
+      message: 'What content entries do you want to migrate?',
+    })
+    .then((name) => name as string)
+    .catch((err) => {
+      cliux.error('Failed to collect the preference');
+      process.exit(1);
+    });
+
+  return strategy;
```

Comment:
There's no necessity to await and store inquire in a variable. Instead, you can directly return the inquire. Because we don't do anything inside the then function.